web/gui/dashboard_info.js

```diff
@@ -2625,5 +2637,17 @@ netdataDashboard.context = {
             '<code>2</code>: non-security updates are available; ' +
             '<code>3</code>: security updates are available; ' +
             '<code>4</code>: an error retrieving information on software updates.'
+            },
+
+    // ------------------------------------------------------------------------
+    // Zookeeper
+
+    'zookeeper.server_state': {
+        info:
+            '<code>0</code>: unknown; ' +
+            '<code>1</code>: leader; ' +
+            '<code>2</code>: follower; ' +
+            '<code>3</code>: observer; ' +
+            '<code>4</code>: standalone.'
```

Comment:
From 0 to 3 you finished the line using semi-colon (`;`) , but for the last code you are finishing the line using  dot(`.`), Jasek already showed me that JS has been more flexible nowadays, but I confess I never heard nothing about to finish a command line with dots, is this acceptable nowadays?

Reply:
it is just a text

Reply:
All right, thank you!